components/locale-chooser.js

```diff
@@ -57,3 +130,40 @@ function getLocaleAndHost() {
     return {locale: fallbackLocale, host}
   }
 }
+
+function mapLocale(key) {
+  const svgStyle = {width: '1em', height: '100%'}
+  switch (key) {
+    case 'en':
+      return {
+        key,
+        display: 'English',
+        flag: <EnSvg {...svgStyle} />,
+      }
+    case 'es':
+      return {
+        key,
+        display: 'Español',
+        flag: <EsSvg {...svgStyle} />,
+      }
+    case 'fr': {
+      return {
+        key,
+        display: 'Français',
+        flag: <FrSvg {...svgStyle} />,
```

Comment:
Ah, you know what, I'll bet that the svg plugin we're using doesn't support require statements. Let's just stick with what you've got. It's not really ergonomic, but it's ok.